content/workload-reduction-toolkit/evaluate-workload-measures/evaluate-progress-with-staff.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Evaluate progress with staff
+layout: /markdown_resource.html.erb
```

Comment:
I made it so that the markdown_resources layout is default for markdown pages, so we don't need these anymore! Looks like there was a rogue on leftover in `content/workload-reduction-toolkit/evaluate-workload-measures/staff-workload-survey.md`

```suggestion
```